docs/tutorials/index.rst

```diff
@@ -82,6 +82,12 @@ Tabular Prediction
 
       How to predict multiple columns in a data table.
 
+   .. card::
+      :title: Adding a Custom Model
+      :link: tabular-custom-model.html
+
+      How to add a custom model to AutoGluon.
```

Comment:
custom tabular model

Reply:
With the shift of AbstractLearner and AbstractTrainer to core, I think this will be more relevant to adding custom models  in general, not just tabular.